lib/postgresql_cursor/active_record/relation/cursor_iterators.rb

```diff
@@ -42,24 +42,26 @@ def each_instance(options={}, &block)
           cursor.iterate_type(self)
         end
 
-        # Plucks the column names from the rows, and return them in an array
+        # Plucks the specified column(s), and return them in an array
         def pluck_rows(*cols)
           options = cols.last.is_a?(Hash) ? cols.pop : {}
           options[:connection] = self.connection
-          self.each_row(options).pluck(*cols)
+          PostgreSQLCursor::Cursor.new(to_pluck_sql(cols), options).pluck(*cols)
         end
         alias :pluck_row :pluck_rows
-
-        # Plucks the column names from the instances, and return them in an array
-        def pluck_instances(*cols)
-          options = cols.last.is_a?(Hash) ? cols.pop : {}
-          options[:connection] = self.connection
-          self.each_instance(options).pluck(*cols)
-        end
-        alias :pluck_instance :pluck_instances
+        alias :pluck_instance :pluck_rows
+        alias :pluck_insntaces :pluck_rows
```

Comment:
Actually `pluck_insntace` and `pluck_instances` return each specified column values as Array, so I think it is the same with `pluck_rows`